Q/QPALM/bundled/patches/freebsd.patch

```diff
@@ -0,0 +1,47 @@
+diff --git a/include/util.h b/include/util.h
```

Comment:
It'd be good to upstream this patch

Reply:
Will do. It's good to be optimistic, but after several years, the authors still haven't made a proper release.